packages/nimble-components/src/table/components/group-row/styles.ts

```diff
@@ -74,6 +75,7 @@ export const styles = css`
 
     .group-row-child-count {
         padding-left: 2px;
+        padding-right: calc(${standardPadding} / 2);
```

Comment:
This is just to give the text of the group row header an 8px margin with the right side of the table.